bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -920,6 +920,43 @@ private String loginCozytouch()
         }
     }
 
+    private void loginOAUTH() throws InterruptedException, TimeoutException, ExecutionException, JsonSyntaxException {
+        String authBaseUrl = "https://" + SOMFY_OAUTH2_URL;
+
+        String urlParameters = "client_id=" + SOMFY_OAUTH2_CLIENT_ID + "&client_secret=" + SOMFY_OAUTH2_CLIENT_SECRET
```

Comment:
I am surprised, why do you use hardcoded client id & secret ?
Normally, they are configuration parameters specific to a user that he should provide.
You cannot share your id/secret, in case it is what you have done.

Reply:
these are not constants related to a specific user, neither mine. It was sniffed from the classic.tahomalink.com portal communication. A user always authenticates using his credentials, the combination of this client id & secret just identifies the Classic Tahoma portal.

Reply:
Ok